File: Serene/Serene.Script/Meeting/MeetingDecisionRelevant/MeetingDecisionRelevantGrid.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System;
    using System.Collections.Generic;
    using System.Runtime.CompilerServices;

    [ColumnsKey("Meeting.MeetingDecisionRelevant"), IdProperty("MeetingDecisionRelevantId"), IsActiveProperty("IsActive")]
    [DialogType(typeof(MeetingDecisionRelevantDialog)), LocalTextPrefix("Meeting.MeetingDecisionRelevant"), Service("Meeting/MeetingDecisionRelevant")]
    public class MeetingDecisionRelevantGrid : EntityGrid<MeetingDecisionRelevantRow>
    {
        public MeetingDecisionRelevantGrid(jQueryObject container)
            : base(container)
        {
        }
    }

    // Please remove this partial class or the first line below, after you run ScriptContexts.tt
    [Imported, Serializable, PreserveMemberCase] 
    public partial class MeetingDecisionRelevantRow
    {
    }
}

File: Serene/Serene.Web/Modules/Meeting/MeetingDecisionRelevant/MeetingDecisionRelevantGrid.ts

namespace Serene.Meeting {
    
    @Serenity.Decorators.registerClass()
    export class MeetingDecisionRelevantGrid extends Serenity.EntityGrid<MeetingDecisionRelevantRow, any> {
        protected getColumnsKey() { return 'Meeting.MeetingDecisionRelevant'; }
        protected getDialogType() { return MeetingDecisionRelevantDialog; }
        protected getIdProperty() { return MeetingDecisionRelevantRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingDecisionRelevantRow.localTextPrefix; }
        protected getService() { return MeetingDecisionRelevantService.baseUrl; }

        constructor(container: JQuery) {
            super(container);
        }
    }
}